src/api/cristinApi.ts

```diff
@@ -80,6 +80,21 @@ export const fetchPerson = async (personId: string) => {
   return fetchPersonResponse.data;
 };
 
+export const searchForPerson = async (results: number, page: number, name: string) => {
+  const searchParams = new URLSearchParams();
+  searchParams.set('results', results.toString());
+  searchParams.set('page', page.toString());
+  searchParams.set('name', name);
```

Comment:
Disse skulle kankje vært wrappet i hver sin `if`? Ser for med at `name` kan mangle bl.a. og da blir det `&name=` i kallet :grimacing: